src/main/java/org/broadinstitute/consent/http/db/DarCollectionSummaryDAO.java

```diff
@@ -140,7 +140,8 @@ List<DarCollectionSummary> getDarCollectionSummariesForSO(
       "ON dar.reference_id = dd.reference_id " +
     "WHERE c.create_user_id = :userId " +
     	"AND (e.latest = e.electionid OR e.electionid IS NULL) " +
-        "AND (LOWER(data->>'status') != 'archived' OR data->>'status' IS NULL ) "
+        "AND (LOWER(data->>'status') != 'archived' OR data->>'status' IS NULL ) " +
+        "AND (EXISTS (SELECT * FROM data_access_request WHERE (collection_id = c.collection_id and draft = false)))"
```

Comment:
Super minor optimization suggestion ... the select here could be a lot smaller, you don't need all fields. I _think_ this would work:
```suggestion
        "AND (EXISTS (SELECT 1 FROM data_access_request WHERE (collection_id = c.collection_id and draft = false)))"
```

Reply:
Also, any reason to not use the `DarCollectionSummary.FILTER_DRAFT_DAR_COLLECTIONS` string here?

Reply:
oops! just forgot to good catch